install.sh

```diff
@@ -1,6 +1,12 @@
 #!/bin/bash
 set -u
 
+# Check if script is run non-interactively (e.g. CI)
+# If it is run non-interactively we should not prompt for passwords.
+if [[ ! -t 0 || -n "${CI-}" ]]; then
```

Comment:
Is there any CI system where `CI` is defined, but `test -t 0` exits successfully? If so, I already hate that CI system.

Reply:
To put that more succinctly, is `if [[ ! -t 0 ]]; then` sufficient?

Reply:
It most likely is sufficient to only check for `! -t 0`, but later in the code there is a check like this:

```
if [[ -t 0 && -z "${CI-}" ]]; then
  wait_for_user
fi
```

So, I thought, I'll be consistent...

Reply:
@sjackman @hwalinga Given that I've pushed 6f37ca9 to use `NONINTERACTIVE` in more places.